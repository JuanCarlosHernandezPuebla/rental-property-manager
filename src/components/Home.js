import React, { Component } from 'react';
import SearchForm from './Forms/SearchForm';

class Home extends Component {

  render() {
    return (
      <div className="container main-content">
        <h1>Home</h1>
        <p>Under Construction...</p>
        <div>
          <img className="img-fluid" src="/images/home-real-estate.jpg" alt="Home real estate" width="400" height="400" />
        </div>
        <div>
          <SearchForm />
        </div>
      </div>
    );
  }

}

export default Home;