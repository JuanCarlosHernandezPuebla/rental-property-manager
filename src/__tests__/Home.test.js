import React from 'react';
import Home from '../components/Home';
import renderer from 'react-test-renderer';

test('Home render test', () => {
    const component = renderer.create(
        <Home/>
    );
    expect(component.toJSON()).toMatchSnapshot();
});