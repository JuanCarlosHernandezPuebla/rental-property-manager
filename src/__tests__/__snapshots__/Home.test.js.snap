// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home render test 1`] = `
<div
  className="container"
>
  <h1>
    Home
  </h1>
  <p>
    Under Construction...
  </p>
</div>
`;
